be/src/storage/lake/location_provider.h

```diff
@@ -69,6 +69,10 @@ class LocationProvider {
         return join_path(segment_root_location(tablet_id), segment_name);
     }
 
+    std::string del_location(int64_t tablet_id, std::string_view del_name) const {
+        return join_path(segment_root_location(tablet_id), del_name);
```

Comment:
not sure if list operation is needed for the del file, maybe put into a separate directory similar to `txn_log_root_location` & `metadata_root_location`?